index.html

```diff
@@ -10,10 +10,54 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="header">
+        <!-- navigation bar -->
+        <nav class="nav-bar">
+            <img src="img/karma-logo.svg" alt="">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" class="nav-link">Meet Karma</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">How it works</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Store</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Blog</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Help</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Login</a></li>
+            </ul>
+        </nav>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+         <!-- intro section  -->
+        <section class="intro">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+            <button class="learn-btn">Learn More</button>
+        </section>
 
+        <main class="main">
```

Comment:
Hi Anthony your layout is very neat, just maybe try using more descriptive names for your class.